### ISSUE TITLE
gnupg in ming64 broken

### ISSUE THREAD

**Issue by ekigwana** [Alexpux/MINGW-packages#4716]
**mingw-w64 binary**
$ gpg -k
gpg: NOTE: THIS IS A DEVELOPMENT VERSION!
gpg: It is only intended for test purposes and should NOT be
gpg: used in a production environment or with production keys!

Note: I cloned the msys repo and built gnupg but got the same result.

$ grep development_version src/gnupg-2.2.11/ -R
src/gnupg-2.2.11/configure:development_version=yes

Looks like the src preparation does not deal with this.

$ which gpg 
/mingw64/bin/gpg

**msys2 binary**
$ /usr/bin/gpg -k
/home/ekigwana/.gnupg/pubring.kbx
pub   rsa4096 2018-06-21 [SCEA] [expires: 2020-06-22]

The problem shows up if you install gpgme from mingw because it gets picked up in the path first.

Basically, need to add ```sed 's/development_version=yes/development_version=no/' -i configure``` to prepare() step in PKGBUILD as was done for msys2 [here](https://github.com/Alexpux/MSYS2-packages/blob/master/gnupg/PKGBUILD).

**Comment by Alexpux** [comment]
Fixed
